modules/ROOT/pages/sources-reference.adoc

```diff
@@ -0,0 +1,69 @@
+= Mule Sources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, source, sources, redelivery, policy
+
+Sources are a very important part of any Mule Application. When added to a flow, they serve as the entry point for information,
+linking any external communication protocol with internal Mule processing.
+
+Examples of sources are:
+
+* HTTP:Listener
+* FILE:Inbound-endpoint
+* AMQP:Consume
+
+== Configuring Sources
+
+Even though each source implementation will have different configuration parameters, there are some common ones worth mentioning.
+
+=== Redelivery Policy
+
+Each source can be configured with a redelivery policy. That way, an user can add some logic on how to handle messages that arrive at the source multiple times and cause exceptions.
+
+For example:
+[source,xml]
+----
+<http:listener path="exhaustRedelivery" config-ref="listenerConfig">
+    <redelivery-policy maxRedeliveryCount="${maxRedeliveryCount}" idExpression="#[vars.idExpression]"/>
```

Comment:
use an actual count and more usual id expression, such as `payload.transaction.id` or anything more relatable to an end user